data/websites.json

```diff
@@ -504,7 +504,13 @@
       "name": "Tilottama Municipality",
       "url": "https://www.tilottamamun.gov.np/en",
       "description": "Official Website Of Tilottama Municipality"
+    },
+    {
+      "name": "District Administration Office of Rupandehi",
+      "url": "https://daorupandehi.moha.gov.np/en",
+      "description": "Official Website Of   District  Administration office of Rupandehi"
```

Comment:
"description": "Official Website of District Administration Office of Rupandehi"